RadixWallet/Prelude/Functions.swift

```diff
@@ -21,7 +21,27 @@ public func identity<T>(_ t: T) -> T {
 	t
 }
 
-/// Wherever you have to handle `Never`
-public func absurd<T>(_ never: Never) -> T {
-	switch never {}
+public func generateElements<Element>(
+	start: Element,
+	step: (Element) -> Element,
+	count: Int,
+	shouldInclude: (Element) -> Bool
+) -> OrderedSet<Element> where Element: Hashable {
+	var next = start
+	var elements: OrderedSet<Element> = []
+	while elements.count != count {
```

Comment:
can you come up with anything more elegant than `while`? I could not really... not since the inclusion of elements is conditional, so we cannot use a range. And doing something like `(start...start+(count*100)).filter(shouldInclude).prefix(count)` does not work, since `shouldInclude` can return `false` for all, so does not matter if we use some silly arbitrary large multiplier like `100` here..

Reply:
What is this for? It doesn't seem to be used. Not sure what the logic is, is it to generate a certain number of integers with a given step size, but with some gaps?

I guess you could do something like

```swift
var offset = 0
for i in 0..<count {
   // increase the offset while !shouldInclude
}
```